packages/components/src/internal/url/utils.ts

```diff
@@ -5,3 +5,7 @@ export function decodeListResolverPath(resolverPath: string): string {
 export function encodeListResolverPath(containerPath: string): string {
     return ['$CPS', containerPath?.toLowerCase(), '$CPE'].join('');
 }
+
+export function encodeFormDataQuote(key: string): string {
+    return key?.replaceAll('%22', '%2522').replaceAll('"', '%22');
+}
```

Comment:
should definitely add some jest tests for this function.
also maybe add a comment about needing to replace %22 first and then the double quote
and add a comment with the issue number

Reply:
jest tests added.